testsettings.py

```diff
@@ -2,6 +2,12 @@
 DRIP_FROM_EMAIL = ''
 DEBUG = True
 
+"""
+To disable the library from sending emails and send it yourself using
+the post_drip signal.
+"""
+SEND_DEFAULT_DRIP_EMAIL = True
```

Comment:
`SEND_DEFAULT_DRIP_EMAIL` it makes me feel it will send a default email
can we call it `DRIP_CAMPAIGN_DRYRUN` and by default it will be `False`